lib/cadet_web/router.ex

```diff
@@ -41,11 +40,12 @@ defmodule CadetWeb.Router do
       info: %{
         version: "1.0",
         title: "cadet"
-      }
+      },
+      basePath: "/v1"
     }
   end
 
-  scope "/api/swagger" do
+  scope "/v1/swagger" do
```

Comment:
Typically we don't scope the swagger doc with API version because it's supposed to cover all versions.